draft-ietf-mls-architecture.md

```diff
@@ -669,7 +672,7 @@ versions and ciphersuites. As such, there may be multiple KeyPackages stored by
 each user for a mix of protocol versions, ciphersuites, and end-user devices.
 
 When a client wishes to establish a group or add clients to a group, it first
-contacts the DS to request KeyPackages for each other client,
+contacts the DS to request KeyPackages for the other clients,
```

Comment:
```suggestion
contacts the DS to request a KeyPackage for each other client,
```

Emphasizes that you need a KeyPackage for each other client.

Reply:
I used "each of the other clients"